torch/csrc/jit/pybind_utils.h

```diff
@@ -250,31 +250,29 @@ inline py::object toPyObject(IValue&& ivalue) {
 inline Stack createStackForSchema(
     const FunctionSchema& schema,
     py::args args,
+    size_t arg_start,
```

Comment:
This is soooo ugly 😞

Reply:
Agreed, but the alternatives are worse:

1. slicing the tuple requires allocating a new tuple, which is added work along the invocation pathway. 
2. upgrading pybind11 in tp2 is really tedious.

Reply:
I agree we don't want to allocate a temporary in there. It's just pretty sad that some random fbcode requirements are forcing us to make the code this much worse 😕